packages/docsite/src/pages/docs/learn/getting_started.md

```diff
@@ -0,0 +1,314 @@
+---
+sidebar_position: 1
+slug: /
+---
+
+# Getting Started
+
+Helium Program Library lets you build on the Helium Network.
+
+Looking to learn more about the Helium Program Library? You're in the right place!
+
+## Initializing the SDK
+
+Let's get started by installing the sdks
+
+```shell
+yarn add @helium/account-fetch-cache @helium/account-fetch-cache-hooks @helium/address @helium/crypto-react-native @helium/currency @helium/currency-utils @helium/data-credits-sdk @helium/distributor-oracle @helium/helium-entity-manager-sdk @helium/helium-react-hooks @helium/helium-sub-daos-sdk @helium/hotspot-utils @helium/http @helium/idls @helium/lazy-distributor-sdk @helium/spl-utils
```

Comment:
This isn't all the packages. Why this set of packages?

Reply:
I removed and added some basic ones